plugins/cockroachdb_logs.yaml

```diff
@@ -0,0 +1,260 @@
+version: 0.0.1
+title: CockroachDB
+description: Log parser for CockroachDB
+parameters:
+  - name: enable_health_log
+    description: Enable to collect health logs
+    type: bool
+    default: true
+  - name: health_log_path
+    description: The absolute path to the CockroachDB health logs
+    type: "[]string"
+    default:
+      - "/var/log/cockroach-data/logs/cockroach-health.log"
+  - name: enable_dev_log
+    description: Enable to collect general developer logs.
+    type: bool
+    default: true
+  - name: dev_log_path
+    description: The absolute path to the CockroachDB Dev Logs
```

Comment:
Should refer to these as dev logs or developer logs consistently between the different parameters.